### PR TITLE
CORE-413 Make Weakly Encrypted encrypted again

### PR DESCRIPTION
**Pull request by ghost**
What do you think of this micro change? Do you see any unwanted side effects?

### CODE REVIEW

File: src/message_api.c [hunk 1]
```diff
@@ -2908,8 +2908,8 @@ static PEP_STATUS encrypt_message_possibly_with_media_key(
     }
         
     if (enc_format == PEP_enc_none || !dest_keys_found ||
-        stringlist_length(keys)  == 0 ||
-        _rating(max_comm_type) < PEP_rating_reliable)
+        stringlist_length(keys) == 0 ||
+        (_rating(max_comm_type) < PEP_rating_reliable && max_comm_type != PEP_ct_OpenPGP_weak_unconfirmed))
```
Comment (ip-plank-security) [inline src/message_api.c:2912]: wouldn't this be sufficient? 

_rating(max_comm_type) < PEP_ct_OpenPGP_weak_unconfirmed

Comment (ghost) [reply src/message_api.c:2912]: That could be comparing `PEP_rating` with `PEP_comm_type`.

Comment (ip-plank-security) [reply src/message_api.c:2912]: ah, I see. in that case - your change makes sense.

**Review by ip-plank-security — APPROVED** [review]

**Review by leviathanch — APPROVED** [review]
Weak encryption is better than no encryption at all.
Approved from my side.

**Pull request merged**